src/components/UsercentricsProvider.tsx

```diff
@@ -2,12 +2,9 @@ import { FC, ReactNode, useMemo } from 'react'
 import React, { useCallback, useEffect, useState } from 'react'
 
 import { UsercentricsContext } from '../context.js'
-import type { UCUICMPEvent, UCWindow } from '../types.js'
-import { UCUICMPEventType } from '../types.js'
+import { ServiceState, UserCentericsService } from '../services/user-centrics-service.js'
```

Comment:
This is nitpick, but let's normalize the capitalization to `Usercentrics`, since that's how the company name is written.

Reply:
yep, fixed.